test/environment_test.go

```diff
@@ -63,6 +63,17 @@ func (s *CLITestSuite) TestEnvironmentUse() {
 	}
 }
 
+func (s *CLITestSuite) TestEnvironmentUpdate_PackageDowngrade() {
+	tests := []CLITest{
+		{args: "environment update env-595 --governance-package essentials", fixture: "environment/sg-downgrade-fail.golden", exitCode: 1},
```

Comment:
Should we also test the case where the downgrade is successful?

Reply:
Downgrade should never be successful :)

Reply:
Should I add a comment to clarify that?

Reply:
You could update the description of the `--governance-package` flag to say that!